code/game/objects/items/oddities.dm

```diff
@@ -80,7 +80,7 @@
 
 /obj/item/weapon/oddity/common/photo_coridor
 	name = "surreal maint photo"
-	desc = "The corridor in this photograph looks familiar, though something seems wrong about it; as if everything in it was replaced with an exact replica of itself."
+	desc = "The corridor in this photograph looks familiar, though something seems wrong about it; It's as if everything in it was replaced with an exact replica of itself."
```

Comment:
But, a semi-colon doesn't mean you are starting a new sentence, it's for a pause longer than a comma. The previous itteration still made sense.